### PR TITLE
Add fkw talk

### PR DESCRIPTION
**Pull request by sfiligoi**

### CODE REVIEW

File: _data/people/fkw888.yml [hunk 1]
```diff
@@ -6,6 +6,12 @@ institution: University of California, San Diego
 website: https://www-physics.ucsd.edu/fac_staff/fac_profile/faculty_description.php?person_id=494
 photo: /assets/images/team/Frank-Wuerthwein.jpg
 presentations:
+  - Title: "Advancing Open Science through distributed High Throughput Computing"
```
Comment (osg-cat) [inline _data/people/fkw888.yml]: Does the system that processes this data treat `Title` the same as `title`? I would probably change this label to lowercase, just to be sure.

File: _data/people/fkw888.yml [hunk 2]
```diff
@@ -6,6 +6,12 @@ institution: University of California, San Diego
...
+    date: 12 Apr 2019
+    url: https://drive.google.com/file/d/1M8w5ndUURkrsDDomhPFpGco4e0efHCVq/view?usp=sharing
+    meeting: "UCSD CI Engineering Brwon Bag"
```
Comment (osg-cat) [inline _data/people/fkw888.yml]: Brown

File: _data/people/fkw888.yml [hunk 3]
```diff
@@ -6,6 +6,12 @@ institution: University of California, San Diego
...
+    meetingurl: https://drive.google.com/drive/folders/1jrjmOGvuVA6RsxvkNFlR1dLXc_t8r7Od
```
Comment (osg-cat) [inline _data/people/fkw888.yml]: This URL is not obviously about the meeting. I would omit this line, unless you can find a human-readable web page that describes or is about the brown bag series or the group that runs it.

**Review by osg-cat — CHANGES_REQUESTED** [review]

File: _data/people/fkw888.yml (hunk 1 above)
Comment (sfiligoi) [reply _data/people/fkw888.yml]: Fixed

File: _data/people/fkw888.yml (hunk 2 above)
Comment (sfiligoi) [reply _data/people/fkw888.yml]: Fixed

**Review by osg-cat — APPROVED** [review]

**Pull request merged**